lib/psabpf_table.c

```diff
@@ -497,6 +567,16 @@ int psabpf_matchkey_prefix(psabpf_match_key_t *mk, uint32_t prefix)
     return NO_ERROR;
 }
 
+/* only for lpm */
+uint32_t psabpf_matchkey_get_prefix(psabpf_match_key_t *mk)
```

Comment:
This method returns `prefix_len` so I would name it `...get_prefix_len`

Reply:
There is also function `psabpf_matchkey_prefix`, so it also should be renamed to achieve symmetry.

Reply:
Done